control-plane-operator/controllers/hostedcontrolplane/assets/cluster-bootstrap/cluster-infrastructure-02-config.yaml

```diff
@@ -16,6 +16,8 @@ status:
   etcdDiscoveryDomain: {{ .BaseDomain }}
   infrastructureName: "{{ .InfraID }}"
   platform: {{ .PlatformType }}
+  controlPlaneTopology: {{ .ControllerAvailabilityPolicy }}
```

Comment:
controlPlaneTopology should be `External` (not dependent on ControllerAvailabilityPolicy)

Reply:
`External` is added in 4.9? currently this branch is using 4.8, so doesn't have that field 😢

Reply:
Just leave it unset for now then

Reply:
https://github.com/openshift/api/blob/release-4.8/config/v1/types_infrastructure.go#L86
just tried. seems the field doesn't omitempty 😵 and it cannot be applied with blank?
```
2021-09-07T18:11:04.475Z	INFO	hypershift-operator.InfraStatus	Updating infrastructure status
2021-09-07T18:11:04.481Z	ERROR	controller-runtime.manager.controller.infra-status	Reconciler error	{"name": "cluster", "namespace": "", "error": "Infrastructure.config.openshift.io \"cluster\" is invalid: status.controlPlaneTopology: Unsupported value: \"\": supported values: \"HighlyAvailable\", \"SingleReplica\""}
```

shall we always give it a default value? e.g. HighlyAvailable/SingleReplica

Reply:
HighlyAvailable is fine